src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -94,64 +94,4 @@ public void deflateInflateWithIntelDeflater() {
             Assert.assertEquals(input, result);
         }
     }
-
-    @Test(enabled = false)
-    public void bamWriteReadWithIntelDeflater() throws IOException {
```

Comment:
Can you explain why this test was deleted, and why it was marked as `enabled = false`?